frontend/src/components/UserSearch.vue

```diff
@@ -96,16 +96,19 @@
             v-model="organizationInput"
             :items="organizations"
             item-value="id"
-            dense
             outlined
+            dense
         ></v-autocomplete>
       </v-col>
       <v-col class="col-6">
         <label for="adv-search-role">
           Role
         </label>
-        <v-text-field
+        <v-autocomplete
             id="adv-search-role"
+            v-model="roleInput"
+            :items="clientRoles"
+            item-value="id"
```

Comment:
It's very difficult to clear the value after choosing something. If you delete the field contents and click/tab out of the field, the contents reappear. The only thing I found that works it to delete the field contents and then press "Esc".

Reply:
🐛

There also seems to be a bug here. If I enter a role, e.g. "GIS : ADMIN", and run a search, I get results. But then if I delete the field contents (delete "GIS : ADMIN" and press Esc), and then re-reun the search, no search is executed: no request is sent, but the datatable shows the loading indicator. If I then enter a role, I can search successfully again.

Reply:
Added empty entry at the beginning of the Role dropdown to clear the value of `roleInput`. The fix should address this bug.